region/maridia/inner-yellow.json

```diff
@@ -1691,36 +1721,107 @@
+                      "type": "contact",
+                      "hits": 1
+                    }}
+                  ]
+                },
+                {
+                  "name": "Screw Kill",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "SpaceJump"
+                    ]},
                     {"or": [
-                      "canPreciseWalljump",
-                      {"enemyDamage": {
-                        "enemy": "Choot",
-                        "type": "contact",
-                        "hits": 1
+                      {"enemyKill":{
+                        "enemies": [ [ "Choot" ] ],
+                        "explicitWeapons": ["ScrewAttack"]
                       }},
+                      {"and": [
+                        {"or": [
+                          "SpaceJump",
+                          "canWalljumpWithCharge"
+                        ]},
+                        {"enemyKill":{
+                          "enemies": [ [ "Choot" ] ],
+                          "explicitWeapons": ["PseudoScrew"]
+                        }}
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Carefully Kill Choot",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "h_canFly"
+                    ]},
+                    {"or": [
+                      "canTrickyJump",
+                      {"and": [
+                        "h_canIBJ",
+                        "canBombHorizontally",
+                        "canCarefulJump"
+                      ]},
+                      {"and": [
+                        "canCarefulJump",
+                        {"enemyKill":{
+                          "enemies": [ [ "Choot" ] ],
+                          "explicitWeapons": [
+                            "Missile",
+                            "Super"
+                          ]
+                        }}
+                      ]}
+                    ]}
+                  ],
+                  "note": [
+                    "Shoot the Choot either before landing or from the safety of the ledge edge.",
+                    "Ending an IBJ by bombing into the wall will safely put Samus on the ledge."
+                  ]
+                },
+                {
+                  "name": "Kill Choot and Reclimb",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "h_canFly"
+                    ]},
+                    {"or": [
                       {"enemyKill":{
                         "enemies": [ [ "Choot" ] ],
                         "explicitWeapons": [
```

Comment:
With this as a strat now, I wonder if it should be in logic with no items - power beam kill.